src/e_lgammaf_r.jl

```diff
@@ -0,0 +1,105 @@
+#=
+/* e_lgammaf_r.c -- float version of e_lgamma_r.c.
```

Comment:
I'm not sure if following the C file naming conventions is what we want.

Reply:
Place both implementations in single file `logabsgamma.jl` ?

Reply:
I think that makes sense. The other option would be to put these in a folder. I don't really know what the best option is here.

Reply:
A separate folder makes preservation of the respective copyrights simpler (or, harder to lose once this is merged).